tkg/web/src/app/views/landing/aws-wizard/aws-wizard.module.ts

```diff
@@ -1,7 +1,7 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 
-import { WizardSharedModule } from './../wizard/shared/wizard-shared.module';
```

Comment:
Are these supposed to be changed? I don't see why this PR would want to change anything in tkg/web/

Reply:
I was replacing all occurrences of `./../` with `../` across the project. That's how these files got changed. Can revert if necessary.